src/pages/home/report/ReportActionItemDraft.tsx

```diff
@@ -1,22 +1,21 @@
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, {ReactNode} from 'react';
 import {View} from 'react-native';
 import useThemeStyles from '@styles/useThemeStyles';
 
-const propTypes = {
+type ReportActionItemDraftProps = {
     /** Children view component for this action item */
-    children: PropTypes.node.isRequired,
+    children: ReactNode;
```

Comment:
I think you can use [ChildrenProps](https://github.com/Expensify/App/blob/main/src/types/utils/ChildrenProps.ts) type instead

Reply:
updated